src/session-adapters/cookieAdapter.ts

```diff
@@ -19,41 +20,49 @@ const createScopedKey = ({
 // We do not use `clientId` in cookie adapter,
 // because different plugins will have different domain names,
 // and it's enough to differentiate these cookie values.
+
+const sessionKey = 'sb.auth'
+//NOTE: possibly cookieAdapter can become createCookieAdapter(key: string)
```

Comment:
WDYT about this comment @eunjae-lee ?

Reply:
if you agree with this I will also need to change a bit the docs (just a note for myself)

Reply:
Yeah agree with create... naming

Reply:
on it

Reply:
https://github.com/storyblok/app-extension-auth/pull/42/commits/ff31f6712d500c35587050508b9707788d7d121f